### DISCUSSION TITLE
Home Assistant

### DISCUSSION THREAD

**Discussion by Microleito** [Dilbert66/esphome-vistaECP#111] [Ideas]
Hello, I'm Leandro, this is my contribution, I migrated from Polymer to Lit for those who use home assistant.
Please: improve the code structure, review it, add key sounds and publish it.

I am not an expert programmer.

I hope it helps you.

Greetings.
[alarm-keypad-card.zip](https://github.com/Dilbert66/esphome-vistaECP/files/12775219/alarm-keypad-card.zip)

**Comment by Dilbert66** [comment]
This migration was already done in the dev branch. There is no need for this change. Thank you.
